apps/game/src/sandbox/dashboards/trip_table.rs

```diff
@@ -698,3 +715,37 @@ fn make_table_unfinished_trips(app: &App) -> Table<App, UnfinishedTrip, Filters>
 
     table
 }
+
+
+fn export_trip_table(app: &App) -> Result<String, anyhow::Error> {
```

Comment:
Common shorthand is to `use anyhow::Result` above and just return `Result<String>` here, equivalent to `std::result::Result<String, anyhow::Error>`. The purpose of the anyhow crate is to not care about the error type